pkg/node/devnode.go

```diff
@@ -462,6 +474,7 @@ func NewDevBee(logger log.Logger, o *DevOptions) (b *DevBee, err error) {
 func (b *DevBee) Shutdown() error {
 	var mErr error
 
+	b.quitChunk <- true
```

Comment:
You can `close(b.quitChunk)` instead of passing value here. It does the same thing.

Reply:
But we have to send a signal to goroutine to close it. If we close it without passing a signal then the code will panic.

Reply:
So you need to understand how the channels work. Even if you close the channel, you can read from it and it will tell you that it is closed.

```
c := make(chan struct{})
close(c)
// this next operation is okay and it will immediately return as the channel is closed. The second parameter
// tells you that it is closed. But we dont care about it as this channel is only used to signal. So no need to write
// anything in it.
<-c
```

This type of pattern is used all over the code. You can follow any closer method implemented by the packages and see.

Reply:
Thanks for explanation.